STANDARDS.md

```diff
@@ -79,17 +83,16 @@ Each dataset shall be versioned.
 
 * The major version shall indicate the STANDARDS that the dataset conforms to. Datasets which are not intended to conform to any STANDARDS should start with 0, e.g. `"v0.1"`. 
 
-* Datasets with versions starting with `"v1.x"` and `"v2.x"` do not follow any official STANDARDS, and thus should be considered `"v0.x"`.
+* Datasets with versions starting with `"v1.x"` and `"v2.x"` do not follow any official STANDARDS, and thus should be considered `"v0.x"`. 
 
 - A minor version change (e.g. `"v3.1"`) represents a minor addition and/or fixes problems:
-	- Adding molecules
+	- Adding molecules or removing/replacing records with an underlying problem
 	- Adding compute specifications
 	- Errors/bugs in the molecule specification
 	- Changes necessary to adhere to the STANDARDS (i.e. changes necessary to placate the NONE compliance status)
 
 - A micro version change (e.g. `"v3.1.1"`) represents a cosmetic change, or a change that is based on dynamic information that does not change the underlying data:
```

Comment:
Given that we've eliminated the compute blacklist in favor of filtering out problems and bumping the version, do we want to use a micro version? I can't think of a cosmetic change that would require us to specify a version, for example updating the status or description wouldn't need a new version IMO